src/otx/algo/classification/efficientnet_b0.py

```diff
@@ -101,6 +101,9 @@ def load_from_otx_v1_ckpt(self, state_dict: dict, add_prefix: str = "model.model
         """Load the previous OTX ckpt according to OTX2.0."""
         return OTXv1Helper.load_cls_effnet_b0_ckpt(state_dict, "multiclass", add_prefix)
 
+    def _reset_prediction_layer(self, num_classes: int) -> None:
+        return
```

Comment:
why we need this change?

Reply:
If it doesn't exist, training this model doesn't work